src/edit/administrativeNoteFields.js

```diff
@@ -11,10 +11,16 @@ import {
   TextArea,
 } from '@folio/stripes/components';
 
+import {
+  validateFieldLength
+} from '../utils';
+
 const AdministrativeNoteFields = () => {
   const { formatMessage } = useIntl();
 
   const administrativeNoteLabel = formatMessage({ id: 'ui-inventory.administrativeNote' });
+  const START_WITH_MAX_LENGTH = 32000;
```

Comment:
Let's move this constant to `constant.js`, it is used in many paces.

Reply:
ok